StudentList/Repositories/StudentsRepository.cs

```diff
@@ -115,40 +111,35 @@ public StudentsRepository(Activity activity, LoadingDelays loadingDelays)
             return temp.ToList();
         }
 
-        private ValidationResult Validate(SavingPhotoResult photo, string name, string birthdate, string group, string uni, string phone)
+        private ValidationResult Validate(string name, DateTime birthdate, string group, string uni, string phone)
         {
             var validationResult = new ValidationResult();
 
-            if (photo.IsError)
-            {
-                validationResult.Errors.Add(nameof(photo), new List<string> { this.activity.GetString(Resource.String.photo_toast_error) });
-            }
-
             if (string.IsNullOrWhiteSpace(name))
             {
-                validationResult.Errors.Add(nameof(name), new List<string>() { this.activity.GetString(Resource.String.name_field_error) });
+                validationResult.Errors.Add(nameof(name), new List<string>() { "Name cannot be empty" });
```

Comment:
And how you gonna have these strings localized?)

Reply:
![image](https://user-images.githubusercontent.com/26973559/42687086-99f053a2-86a0-11e8-8b3d-d8cd18d71419.png)
Could I write kind of that?

Reply:
I fixed that